cobalt/browser/application.cc

```diff
@@ -212,21 +212,31 @@ GURL GetInitialURL(bool should_preload) {
   GURL initial_url = GURL(kDefaultURL);
   // Allow the user to override the default URL via a command line parameter.
   base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
-  if (command_line->HasSwitch(switches::kInitialURL)) {
-    std::string url_switch =
-        command_line->GetSwitchValueASCII(switches::kInitialURL);
+  bool has_initial_url_switch = command_line->HasSwitch(switches::kInitialURL);
+
+  const base::CommandLine::StringVector& args = command_line->GetArgs();
+  if (has_initial_url_switch || !args.empty()) {
```

Comment:
nit: you only need to check `!args.empty()` for this condition, as it will always be true when `has_initial_url_switch` is true.

Reply:
`CommandLine::GetArgs()` actually does `Get the remaining arguments to the command.`, so args is empty when there are only command-line switches.